internal/pkg/config/testdata/src/configv2/discussion/test-config-v2.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source:
+  value:
+    - id: "Tokens are secret"
+      package: configv2/discussion
+      type: Token
+    - id: "TokenBundles are entirely secret"
+      package: configv2/discussion
+      type: TokenBundle
+    - id: "NamedToken.token is secret, NamedToken.name is not"
+      package: configv2/discussion
+      type: NamedToken
+      fieldName: token
+    - id: "Tagged fields are secret, neighboring untagged fields are not"
+      # empty package and type fields match anything
+      fieldTag:
+      - myTag: source
+    - id: "In context, the parameter of SaveSecret(secret interface{}) is a source"
+      # this could use additional details and perhaps a better name than "scope"
+      scope: parameter
```

Comment:
What if there's more than one parameter? What if it's a local variable? What if it's a global variable?